### PR TITLE
Adding DD windows agent install options

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
Adds DD windows Agent install options + clean all reference to DD windows Agent prior to v5

### Motivation
Eng request

### Preview link
* https://docs-staging.datadoghq.com/gus/windows-clean-up/agent/basic_agent_usage/windows/#starting-and-stopping-the-agent

### CODE REVIEW

File: content/agent/basic_agent_usage/windows.md [hunk 1]
```diff
@@ -19,15 +19,36 @@ further_reading:
...
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
+This page outlines the basic functionality of the Windows Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
 
 ### Starting and Stopping the Agent
 
-The execution of the Agent is controlled by a Windows service.
+The execution of the Agent is controlled by a Windows service. 
+There are many items that can be configured on the command line. Each configuration item is added as an install property to the command line, for instance the following commands install the Agent, and configure the Agent configuration file with the `<DATADOG_API_KEY>`, as well as setting the `<HOSTNAME>` and tags.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/windows.md]: There are many items that can be configured on the command line. Each configuration item is added as an install property to the command line. For instance, the following commands install the Agent, configure the Agent configuration file with the `<DATADOG_API_KEY>`, and set the `<HOSTNAME>` and tags.

File: content/agent/basic_agent_usage/windows.md [hunk 2]
```diff
@@ -19,15 +19,36 @@ further_reading:
...
+| `PROXY_HOST`      | String | If using a proxy, sets your proxy host. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_PORT`      | Number | If using a proxy, sets your proxy port. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                          |
+
+
+### Agent v6 commands
+
+There are a few major changes compare to older Datadog Windows Agent v5:
```
Comment (cswatt) [inline content/agent/basic_agent_usage/windows.md]: r/compare/compared

File: content/agent/basic_agent_usage/windows.md [hunk 3]
```diff
@@ -113,25 +130,7 @@ If you're running on a version older than 5.2 visit the status page in your web
...
-  * Integration configuration:
-`C:\Documents and Settings\All Users\Application Data\Datadog\conf.d\`
-
-For Windows Server 2008, Vista and newer:
-
-  * Agent configuration:
-`C:\ProgramData\Datadog\datadog.conf`
-  * Integration configuration:
-`C:\ProgramData\Datadog\conf.d\`
+Use the Datadog Agent Manager located in the start menu to enable, disable and configure checks. You have to restart the Agent in order for your changes to be applied.
```
Comment (cswatt) [inline content/agent/basic_agent_usage/windows.md]: serial comma
Use the Datadog Agent Manager located in the start menu to enable, disable, and configure checks. Restart the Agent in order for your changes to be applied.

File: content/agent/basic_agent_usage/windows.md [hunk 4]
```diff
@@ -19,15 +19,36 @@ further_reading:
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
+This page outlines the basic functionality of the Windows Datadog Agent. If you haven't installed the Agent yet, instructions can be found [in the Datadog Agent integration page][1].
 
 ### Starting and Stopping the Agent
```
Comment (derekwbrown) [inline content/agent/basic_agent_usage/windows.md]: My read of this is that the header and the content don't match up.
`Starting and Stopping` the agent seems like a run time thing (i.e. after it's installed, how do you start it and stop it).  And, the first line says `the execution of the agent is controlled by a windows service`, which is also run time.  But all of the items after that are about how to install.

File: content/agent/faq/dogstream.md [hunk 5]
```diff
@@ -53,10 +53,7 @@ file in the following format:
     dogstreams: /var/log/web.log:parsers:parse_web
 
 The `parsers:parse_web` portion indicates that the custom Python function lives
-in a package called  parsers in the Agent's `PYTHONPATH`, and the parsers package
-has a function named `parse_web`. The Agent's `PYTHONPATH` is set in the Agent
-startup script, `/etc/init.d/datadog-agent` for Agent versions < 2.0, and in the
-supervisor config for Agent version >= 2.0.
+in a package called  parsers in the Agent's `PYTHONPATH`, and the parsers package has a function named `parse_web`. The Agent's `PYTHONPATH` is set in the Agent startup script, `/etc/init.d/datadog-agent` in the supervisor config for Agent version.
```
Comment (cswatt) [inline content/agent/faq/dogstream.md]: in a package called `parsers` in the Agent's `PYTHONPATH`, and the `parsers` package has a function named `parse_web`. The Agent's `PYTHONPATH` is set in the Agent startup script, `/etc/init.d/datadog-agent`, in the supervisor config for Agent version.

File: content/agent/basic_agent_usage/amazonlinux.md [hunk 6]
```diff
@@ -18,7 +18,7 @@ further_reading:
 
 ## Overview
 
-This page outlines the basic functionality of the Datadog Agent for Amazon linux.  
+This page outlines the basic functionalities of the Datadog Agent for Amazon linux.
```
Comment (derekwbrown) [inline content/agent/basic_agent_usage/amazonlinux.md]: I think `functionality` was correct

File: content/agent/basic_agent_usage/windows.md [hunk 7]
```diff
@@ -19,15 +19,40 @@ further_reading:
...
+| `PROXY_HOST`      | String | If using a proxy, sets your proxy host. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_PORT`      | Number | If using a proxy, sets your proxy port. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
+| `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                          |
+
+## Agent Commands
+
+The execution of the Agent is controlled by a Windows service.
```
Comment (derekwbrown) [inline content/agent/basic_agent_usage/windows.md]: .... controlled by the Windows Service Control Manager.

File: content/agent/proxy.md [hunk 8]
```diff
@@ -205,7 +205,7 @@ Assuming that the supervisor file is found at `<SUP_FILE>`
 sed -i 's/ddagent.py/ddagent.py --sslcheck=0/' <SUP_FILE>
 ```
 
-For the Windows Agent (Starting from Agent version 3.9.2), edit your configuration file `datadog.conf` and add this option:
+For the Windows Agent, edit your configuration file `datadog.conf` and add this option:
```
Comment (derekwbrown) [inline content/agent/proxy.md]: On Agent6 this is `datadog.yaml`

Comment (l0k0ms) [reply content/agent/proxy.md]: You are right, but here it's in the Agent v5 section.

**Pull request merged**